app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -52,6 +53,14 @@ def check_already_authenticated
     redirect_to after_otp_verification_confirmation_url
   end
 
+  def check_aal3_bypass
+    return unless aal3_policy.aal3_configured_and_required?
+    method = two_factor_authentication_method
+    return if AAL3Policy::AAL3_METHODS.include? method
+    aal3_redirect = aal3_redirect_url(current_user)
+    redirect_to aal3_redirect || aal3_required_url
```

Comment:
If `aal3_redirect` is nil (i.e., they don't have a configured aal3 option) then they should be redirected to `two_factor_options_url`.
`aal3_required_url` is the dead-end andI think we can get rid of the aal3_controller all together.

Reply:
OK, so if they get here, they must already have aal3 configured so they should get the redirect to the aal3 login page. We should think about getting rid of the aal_controller in a separate story.